src/main/scala/be/doeraene/sjsreflect/Reflect.scala

```diff
@@ -45,6 +42,13 @@ object Reflect {
   def getClassForName(fqcn: String): Option[Class[_]] =
     classesByName.get(fqcn)
 
+  /**
+    * Get this class and its descendants. Requires the `selectDescendantsClasses` selector
+    * @param fqcn the class's fully qualified name
+    * @return a js.Dictionary of class name -> Class[_]
+    */
+  def getClassAndDescendants(fqcn: String): Dictionary[Class[_]] = classesByName
```

Comment:
My bad. Would you like me to remove the argument and rename to "getAllClassesAndDescendents"?